WooCommerce/Classes/Authentication/Prologue/LoginPrologueViewController.swift

```diff
@@ -1,112 +1,123 @@
 import Foundation
-import UIKit
-import WordPressAuthenticator
+import SwiftUI
 import Experiments
 import protocol WooFoundation.Analytics
 
 
 /// Displays the WooCommerce Prologue UI.
 ///
-final class LoginPrologueViewController: UIViewController {
-
-    private let analytics: Analytics
-    private let featureFlagService: FeatureFlagService
-
-    /// Background View, to be placed surrounding the bottom area.
-    ///
-    @IBOutlet private var backgroundView: UIView!
-
-    /// Container View: Holds up the Button + bottom legend.
-    ///
-    @IBOutlet private var containerView: UIView!
-
-    /// Curved Rectangle: Background shape with curved top edge
-    ///
-    @IBOutlet private weak var curvedRectangle: UIImageView!
-
-    /// The WooCommerce logo on top of the screen
-    @IBOutlet private weak var topLogoImageView: UIImageView!
-
-    // MARK: - Overridden Properties
-
-    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
-        return .portrait
```

Comment:
I removed this part, we don't enforce any orientation now, the Authentication library still enforces `portrait` orientation for phones, but at least now iPads can show the app in landscape.